packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -145,12 +151,30 @@ If a table contains these two attributes: (System Name | Last Sync) you may want
 
```

Comment:
Small thing, but Abi had recommended we stop using "ie" and start using "in other words" instead.
Same thing with "eg", she recommended we use "For example" instead.
I think it was just incase people didn't know what eg or ie meant! I don't think you've used it anywhere else but I would change it here!

Reply:
Good point. It's there from the previous/current version of doc for this. I'll fix it.